srcpkgs/turnstile/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'turnstile'
+pkgname=turnstile
+version=0.1.1
+revision=1
+build_style=meson
+configure_args=" -Ddinit=disabled"
+hostmakedepends="pkg-config scdoc"
+makedepends="pam-devel"
+short_desc="Independent session/login tracker"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="BSD-2-Clause"
+homepage="https://github.com/chimera-linux/turnstile"
+distfiles="https://github.com/chimera-linux/turnstile/archive/refs/tags/v${version}.tar.gz"
+checksum=10512f46cf4e5f57f861e5847e09271cfb1bf9a281a99265e925a367c7b7093d
+
+pre_configure() {
+	vsed -i "s/backend = dinit/backend = none/g" turnstiled.conf.in
```

Comment:
I've been meaning to look into how to go about making a runit backend

Reply:
My goal is to make backends for runit and 66 :)
Obarun has its own user services implementation for 66 but has issues. I tried to design something better using utlogd which is nice but encountered issues - among others it uses utmp which will require an effort to be suported in musl.
turnstile seems a really nice approach to the issue. If it can replace elogind, even better!